systemtest/050-signing.bats

```diff
@@ -12,6 +12,8 @@ function setup() {
     export GNUPGHOME=$TESTDIR/skopeo-gpg
     mkdir --mode=0700 $GNUPGHOME
 
+    export CONTAINERS_SIGNATURE_PASSPHRASE="MakeTomHappy"
```

Comment:
Hum… this is convenient, and it’s not crossing any security boundary AFAICT, but it also puts a copy of that secret data to every single process executed in this shell. Consider Heartbleed, that kind of unnecessary exposure is not desirable.

Of course it’s _possible_ to use the environment variable in a restricted context but it’s also much easier to do just what this code does.

So I think it should at least be _possible_ to pass the passphrase in a file (OTOH then we need to worry about including, or stripping, a trailing newline — probably do what `gpg2 --passphrase-file` does), and I’m not sure if we should allow environment variables at all.

Reply:
Sure, we can restrict it to a file directly.

OTOH I am not sure if it would change much if used insecurely. Exporting a variable as done in the test is nice for testing but absolutely dangerous in production.

I can alter the c/image logic and turn it into `CONTAINERS_SIGNATURE_PASSPHRASE_FILE` if you prefer that.

Reply:
I was primarily thinking a `skopeo copy --passphrase-file=…` option; direct and explicit.

But I don’t feel strongly against a …`_FILE` environment variable. We do support a few environment variables, so, \*shrug\*.

Reply:
@mtrmac @vrothberg Kindly provide an option to add the passphrase directly in the skopeo copy command 

```
  skopeo copy --pinentry-mode=loopback --passphrase ***
```

Reply:
That makes the passphrase visible to _every single user_ on a system via `/proc/*/cmdline`.

I’m not saying there are absolutely no legitimate use cases, but if we ever add this, it would be named something like `--dangerous-publicly-visible-passphrase=`.